CHANGELOG.md

```diff
@@ -2,6 +2,21 @@
 
 All notable changes to this project will be documented in this file.
 
+## [[8.0.0]](https://github.com/iExecBlockchainComputing/iexec-worker/releases/tag/v8.0.0) 2023
+
+### New Features
+* Support Gramine framework for TEE tasks.
+* Bind SMS URL to task.
+### Bug Fixes
+* Remove dataset decryption non-TEE workflow.
+* Purge result files and metadata when task is completed.
+### Quality
+* Improve code quality.
+### Dependency Upgrades
+* Upgrade to Spring Boot 2.6.14.
+* Upgrade to Gradle 7.6.
+* Upgrade OkHttp to 4.9.0.
```

Comment:
This upgrade has been done in all applications and is related to Web3j upgrade in iexec-common.
Should we propagate this note ?

Reply:
Thank you, updated changelog (+ release notes with commit ID) in modules :+1: